### ISSUE TITLE
Content Security Policy header causes refusal of rocket-loader.min.js resource

### ISSUE THREAD

**Issue by suthsc** [DestinyItemManager/DIM#2811]
When loading DIM, I am presented with the following message:
![image](https://user-images.githubusercontent.com/232286/39953238-e2b39ce4-556c-11e8-9403-e1ff70161b2a.png)

In the console, I see two errors. The first is
```
Refused to load the script 'https://ajax.cloudflare.com/cdn-cgi/scripts/dba9ecf7/cloudflare-static/rocket-loader.min.js' because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-eval' 'unsafe-inline' 'report-sample' https://apis.google.com https://www.google-analytics.com https://platform.twitter.com https://cdn.syndication.twimg.com data:".
```
The second is:
```
POST https://sentry.io/api/279673/csp-report/?sentry_key=<<omitted>> 403 (FORBIDDEN)
```

The value of the `content-security-policy` header is
```
content-security-policy: default-src 'none'; script-src 'self' 'unsafe-eval' 'unsafe-inline' 'report-sample' https://apis.google.com https://www.google-analytics.com https://platform.twitter.com https://cdn.syndication.twimg.com data:; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/css https://platform.twitter.com/css/ https://*.twimg.com/; connect-src 'self' https://*.destinyitemmanager.com https://www.google-analytics.com https://www.bungie.net https://reviews-api.destinytracker.net https://db-api.destinytracker.com https://sentry.io/api/279673/store/; img-src 'self' https://*.destinyitemmanager.com https://www.bungie.net https://ssl.google-analytics.com https://www.google-analytics.com https://csi.gstatic.com https://opencollective.com https://syndication.twitter.com https://platform.twitter.com https://*.twimg.com/ data:; font-src 'self' https://fonts.gstatic.com; child-src 'self' https://accounts.google.com https://content.googleapis.com; frame-src  https://accounts.google.com https://content.googleapis.com https://syndication.twitter.com/ https://platform.twitter.com/; object-src 'self'; manifest-src 'self'; report-uri https://sentry.io/api/279673/csp-report/?sentry_key=<<omitted>>
```

The same problem happens when using Chrome or Brave (a chrome derivative) as well as firefox.

**Comment by bhollis** [comment]
Hmm, I don't have any desire to use rocket-loader. Not sure why cloudflare is adding it.

**Comment by bhollis** [comment]
I checked our config and rocket loader is off. I also have no problems loading app.destinyitemmanager.com. How are you accessing DIM?

**Comment by bhollis** [comment]
My guess is you have some other extension trying to inject something. What do you have installed?

**Comment by suthsc** [comment]
Hmm.. It does seem to be something on my side as loading DIM in an incognito window does work. I thought I had tried that, but must not have. Sorry for the false alarm and thanks for looking into it.
